src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_NotificationsScreen.tsx

```diff
@@ -72,19 +113,15 @@ export const Profile_NotificationsScreen = () => {
                 ProfileTexts.sections.settings.linkSectionItems.notifications
                   .pushToggle.subText,
               )}
-              value={
-                permissionStatus === 'granted' &&
-                isConfigEnabled(config?.modes, 'push')
-              }
+              value={isConfigEnabled(config?.modes, 'push')}
```

Comment:
Why is `permissionStatus === 'granted'`` removed? Should this have been `value={pushEnabled}`?